lib/hotwire/spark/engine.rb

```diff
@@ -8,7 +8,7 @@ class Engine < ::Rails::Engine
     config.hotwire.spark = ActiveSupport::OrderedOptions.new
     config.hotwire.spark.merge! \
       enabled: Rails.env.development?,
-      css_paths: File.directory?("app/assets/builds") ? %w[ app/assets/builds ] : %w[ app/assets/stylesheets ],
+      css_paths: %w[ app/assets/builds app/assets/stylesheets ],
```

Comment:
I'm afraid we can't get rid of this to avoid the duplicated signal problem mentioned here https://github.com/basecamp/hotwire-spark/issues/11

Reply:
Oh, I see.
Reverted this change.